src/components/BobaContainer/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import bobalist from './bobalist.jsx';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import './index.css'
+
+const BobaContainer = () => {
+  return (
+    <div className='indBoba'>
+        {bobalist.map((boba) => (
+            <Card key={boba.name} style={{ width: '18rem' }}>
```

Comment:
Please create reusable `<Card />` component

Reply:
same with this one.

Reply:
Please mark resolved conversation if you validated as completed.